i18n/MonotoneMuseum/ko_kr.yml

```
@@ -25,65 +25,538 @@ ko_kr:
+        - |-
+          【범람】
+          바닷물이 급격하게 늘어나 수조에서 넘쳐흐른다. 씬에 등장해있는 바다지기 클래스를 가지지 않는 캐릭터의 【HP】는 즉시 0이 된다. 단, 《대하의 인도》의 효과를 받은 캐릭터와 《수중생물》을 취득한 캐릭터는 3D6점의 【HP】를 소비하는 것으로 효과를 상쇄한다.
+        - |-
+          【소금의 눈】
+          하늘에서 눈처럼 소금이 쏟아진다. 사람들은 처음에는 기뻐하지만, 계속 쏟아지는 소금은 밭을 휩쓸고 사람들에게 해를 입힌다. 소금은 그럼에도 불구하고 멎지 않는다.
+        - |-
+          【민물화】
+          해수가 민물이 된다. 인간 중에는 기뻐하는 자들도 있지만 바다의 생물들에게는 죽음을 의미한다.
+        - |-
+          【해류소실】
+          바다의 해류가 사라진다. 바다지기의 힘을 가지고도 파도를 일으킬 수 없다. 고인 해수는 부패하고 바다의 은혜를 받아 살아가는 이들이 차례차례 병에 걸린다.
+        - |-
+          【무너진 세계】
+          바다 주변 (바다 내부가 아니라 그 외부)에서 폭발적으로 올풀림이 넓어진다. 그 규모는 GM이 결정하지만 머지않아 주민들은 죽음을 맞이할 것이다.
+        - |-
+          【술의 바다】
+          바닷물이 술이 된다. 물을 마신 모든 것은 취해 이성을 잃는다. 그 모습이 마치 암흑기에 멸망한 "나태의 고도"를 연상시킨다.	
+        - |-
+          【차가운 바다】
+          바닷물이 영혼을 얼릴 정도로 차가워진다. 물에 닿은 것은 어둡고 배타적이고 의심암귀하며 서로를 증오하게 된다. 또한 모든 캐릭터는 해수에 닿아있는 한 매 씬 2D6점 【MP】를 잃는다.
+        - |-
+          【소용돌이】
+          거대한 소용돌이가 출현하다. 소용돌이의 중심은 허무와 통하고 있어 바다지기의 힘으로도 소용돌이를 지울 수 없다. 바다 속에 있는 모든 캐릭터는 2D6을 굴려 나온 수 만큼 【HP】를 잃는다. 단, 주사위 눈이 2인 경우 즉시 【HP】가 0이 된다.
+        - |-
+          【바다의 종언】
+          바다를 유지하는 생명력 그 자체가 사라진다. 물은 썩고 물고기는 죽으며 파도는 잠잠해진다. 모든 생명은 죽는다.
+      EDT:
+        name: 영구소실표
+        type: 2D6
+        items:
+        - |-
+          【무자비한 현실】
+          소망도 희망도 이뤄지지 않는다. 가람을 쓰러트리는 것, 그것이 당신의 최선의, 당신에게 허락된 행복이었다. 의식이, 허무에 삼켜져 사라진다....
+          ＞ 다이스의 추가롤은 불가능하다. 가람이 된다.
+        - |-
+          【생명 소실】
+          허무가 당신의 생명을 삼켰다. 육체가 실이 끊긴 꼭두각시 인형처럼 무너져 지면에 쓰러진다. 이것이, 이 인생의 끝이란 말인가...
+          ＞ 당신은 사망한다. PC로써 계속 이용하고 싶은 경우 캐릭터 성장시 [클래스:송장(TM 90p)]을 취득할 것.
+        - |-
+          【영혼 소실】
+           당신의 마음에 허무가 스며든다. 모든게 텅 비어버린듯한 절망. 아직, 당신은 가람이 되지는 않았다. 아직.... 지금은.
+          ＞ 【정신력】을 -[배우레벨]D6 할 것. (하한 1)
+        - |-
+          【육체 소실】
+          가람에게 잘렸는가 올풀림에 삼켜졌는가. 모든 것을 삼키는 허무가 당신의 육체를 집어삼켰다. 육체의 일부분을 잃었지만 당신은 간신히 살아남았다.
+          ＞ 【내구력】을 -[배우레벨]D6 할 것. (하한 1)
+        - |-
+          【인연 소실】
+          소중한 누군가가 있다. 사랑인가 증오인가, 당신의 인생에 새겨진 유일무이한 존재다. 하지만, 추억은 모두 허무에 삼켜져 사라졌다. 설령 상대의 얼굴을 기억하더라도 애끓는 마음은 더는 없다.
+          ＞ 고정화한 파트너 중 한 명을 상실한다. 연극 중에 상실한 고정화 파트너가 있다면 우선적으로 대가로 삼을것.
+        - |-
+          【경험 소실】
+          어째서 실잣는이가 되었는가... 소중한 무언가가 있었을터다. 자신의 인생을 걸만한, 소중한 "무언가"가. 하지만 더는 기억나지 않는다. 두번 다시 기억해내지 못하리라.
+          ＞ 경험을 소실한다. (특성도 사용 불가)
+        - |-
+          【출신 소실】
+          당신이 어디에서 왔는지, 누구였는지. 고향이나 가족과의 추억이 허무 너머로 사라져간다. 이제 영원히, 기억할 수 없겠지.
+          ＞ 출신을 상실한다. (특성도 사용 불가)
+        - |-
+          【허무의 상흔】
+          사지에 격통이 일었다. 당신의 몸은 허무의 상흔이 뚜렷하게 남아, 사라지지 않고, 그 몸을 좀먹는다.
+          ＞ 【행동치】를 -5 할 것（하한 1）
+        - |-
+          【허무의 침식】
+          고동이 불길한 박자로 울린다. 육체가, 영혼이, 침식되어 사라지는 것 같은 끔찍한 감각. 설령 보지 않더라도, 착실하게 당신은 허무로 향하고 있다…
+        ＞ 애프터 플레이에서 기본박탈치를 +3할 것. (상한9). 대가를지불할 수 있는지는 박탈체크시에 확정할 것.
```

This line seems to be indented one level too shallow, resulting in a syntax error. Please fix it.
```suggestion
          ＞ 애프터 플레이에서 기본박탈치를 +3할 것. (상한9). 대가를지불할 수 있는지는 박탈체크시에 확정할 것.
```